modules/aspects/provides/inputs.nix

```diff
@@ -0,0 +1,41 @@
+{ den, withSystem, ... }:
+let
+  description = ''
+    Provides the `flake-parts` `inputs'` (the flake's inputs with system pre-selected)
```

Comment:
👍 You are awesome, I like this a lot, `den.provides.inputs` is an opt-in feature for people who uses flake-parts.

Reply:
Not sure about the name, but I like it.